utkdigitalinitiatives/polk-correspondence-app
# Updated font-size attribute on letter signature to mimic small-caps.

**JIRA Ticket**: [https://jira.lib.utk.edu/projects/PTC/issues/PTC-74](https://jira.lib.utk.edu/projects/PTC/issues/PTC-74)

# What does this Pull Request do?

Decreases font-size on strings wrapped with `<span class="tei-hi2">` for signatures to mimic small caps by request of MDC. Also updates MDC name in introduction to match.

# What's new?

Adding styling attribute for font-size.

# How should this be tested?

- Run `ant` and upload
- Navigate to a letter and see that small-caps are present. 
- Navigate to introduction to see that MDC signature matches letter styling.

# Additional Notes:
Clear browser cache if necessary. *snark*

# Interested parties
@markpbaggett
